db-lib/db/main.rkt

```diff
@@ -44,6 +45,8 @@
   (-> path-string?)]
  [postgresql-password-hash
   (-> string? string? string?)]
+ [uuid?
```

Comment:
`db/main` should not export `uuid?`; it should only be exported from `db/util/postgresql` (also remove require above)